Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM opensuse:leap
+
+ARG GO_VERSION=${GO_VERSION:-1.10.2}
+
+# Get OBS client, plugins and dependencies
+RUN zypper -n install osc-plugin-install vim curl bsdtar git sudo pcre-tools
+RUN curl -OkL https://download.opensuse.org/repositories/openSUSE:Tools/openSUSE_42.3/openSUSE:Tools.repo
```

Comment:
Could you define a variable at the top for this `42.3` version number?

Reply:
Sure!